### DISCUSSION TITLE
Velge navn på bokser

### DISCUSSION THREAD

**Discussion by forteller** [entur/tavla#314] [Ideas]
Nå som man kan ha flere bokser/kolonner som viser samme holdeplass (takk for det, se #81), så er det naturlige neste steget å kunne gi hver kolonne nye navn for å unngå forvirring. Feks vil jeg gjerne kunne kalle én kolonne "Mot sentrum" og en annen "Fra sentrum". 

Jeg trenger ikke å ha navnet på holdeplassen synlig, for jeg vet hvilken holdeplass det er hvis jeg lager en tavle kun for min nærmeste holdeplass.

Takk igjen!

**Comment by draperunner** [comment]
Hei, @forteller! Takk for forslaget. Forstår behovet. Me får tenke litt på korleis dette kan løysast på ein god måte.

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
